### PR TITLE
Cards

### PR DESCRIPTION
**Pull request by BALAJI141031**

### CODE REVIEW

File: pages/components/cards/cards.css [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+.ver-card > div > p {
+  text-align: center;
+}
+.overlay {
+  width: 100%;
+  position: absolute;
+  left: 0em;
+  top: 0em;
+  color: #ffff;
+  background-color: rgb(168, 168, 168);
```
Comment (deekshithmd) [inline pages/components/cards/cards.css]: using color variable will be good

File: pages/components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+  align-items: center;
+  height: max-content;
+  padding: 5px;
+  width: 100%;
+  z-index: 2;
+  left: 50%;
+  top: 50%;
+  transform: translateX(-50%);
+  color: #fff;
+  font-size: 18px;
```
Comment (deekshithmd) [inline pages/components/cards/cards.css]: using rem for font will be good

**Review by deekshithmd — APPROVED** [review]

File: pages/components/inputs/input.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+.success {
+  border-color: palegreen;
+}
+.warn {
+  border-color: rgb(255, 150, 14);
+}
+.error {
+  border-color: red;
+}
```
Comment (nisargpawade) [inline pages/components/inputs/input.css:29]: you can create variables for these colors

**Review by nisargpawade — APPROVED** [review]

File: pages/components/inputs/input.css (hunk 3 above)
Comment (BALAJI141031) [reply pages/components/inputs/input.css:29]: ok

File: pages/components/cards/cards.css (hunk 2 above)
Comment (BALAJI141031) [reply pages/components/cards/cards.css]: ok

**Pull request merged**